src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -212,10 +214,8 @@
     public static final String FINGERPRINT_SUMMARY_FILE_SUFFIX = "fingerprinting_summary_metrics";
     public static final String FINGERPRINT_DETAIL_FILE_SUFFIX = "fingerprinting_detail_metrics";
 
-    // Stock main method
-    public static void main(final String[] args) {
-        new CheckFingerprint().instanceMainWithExit(args);
-    }
+    private Path inputPath;
+    private Path genotypesPath;
```

Comment:
It's hard to see in all the sea of diffs, but could these two be defined in the `doWork` method?